src/EnergyPlus/ConvectionCoefficients.cc

```diff
@@ -6394,7 +6394,7 @@ namespace ConvectionCoefficients {
 			FlowRegimeStack( 0 ) = InConvFlowRegime_A3;
 		} else { // is controlled, lets see by how and if that means is currently active
 
-			if ( ! ( ZoneEquipConfig( ZoneNum ).EquipListIndex > 0 ) ) {
+			if ( ! ( ZoneEquipConfig( ZoneNum ).EquipListIndex > 0 ) || SysSizingCalc || ZoneSizingCalc || ! ZoneEquipSimulatedOnce ) {
```

Comment:
Makes sense to me.  As for whitespace, I don't mind if the ! is right up against the boolean, like !something.  But no worries here.